prime-router/metadata/hl7_mapping/datatypes/dateTime/DTM.yml

```diff
@@ -0,0 +1,7 @@
+elements:
+  - name: dateTime
+    value:
+      - '%resource.extension(%`rsext-hl7v2-date-time`).value'
+      - '%resource.extension("http://hl7.org/fhir/StructureDefinition/patient-birthTime").value'
+      - '%resource'
```

Comment:
@mkalish The last two values here could be seen as logical fallbacks, but they are harmless in HL7->FHIR->HL7 cases, and they prevent us from needing to add transforms for SR and from needing to update basically every FHIR->HL7 test. I suppose the second value here is kind of optional, but I don't see any ambiguity around whether or not a FHIR dateTime's value should be used for the HL7 DTM in the absence of the rsext extension, so making this slight exception to the general approach seemed okay to me. What do you think?

Reply:
Avoiding having to make all the other changes sounds like a good reason to me, but maybe drop a comment around why the fallback exists and what the long term would look like?